docs/data/destinations/meta-pixel.md

```diff
@@ -3,16 +3,18 @@ title: Meta Pixel Event Streaming
 description: Amplitude Data's Meta Pixel integration lets you stream your Amplitude event data straight to Meta Pixel with just a few clicks.
 ---
 
-Amplitude Data's Meta Pixel integration lets you stream your Amplitude event data straight to Meta Pixel with just a few clicks.
-
 --8<-- "includes/open-beta.md"
 
+Amplitude Data's Meta Pixel integration lets you stream your Amplitude event data straight to Meta Pixel with just a few clicks.
+
 ## Considerations
 
-Keep these things in mind when sending Amplitude data to Meta. 
+Keep these things in mind when sending Amplitude data to Meta.
 
-- Amplitude uses the `user_id` to match the `external_id` user property field in Meta Pixel. Make sure that this field is present in your Amplitude events.
-- All event properties that Amplitude sends are stored as 'custom data'. Because of this, normal Meta conversion can't be applied to those events. You must create a custom conversion using this [Meta guide](https://www.facebook.com/business/help/2375212726097833?id=1205376682832142 "https://www.facebook.com/business/help/2375212726097833?id=1205376682832142").
+- Amplitude uses the `external_id` user property field in Meta Pixel for user matching.
+- Amplitude allows you to customize mappings that will appear in the `user_data` array of Meta Pixel for conversion.
+- User data that's sent to MetaPixel that requires hashing will be hashed. Ensure that the data isn't already hashed.
+- Other event properties selected in the **Specify event properties to send** that Amplitude sends are stored as 'custom data'. Because of this, normal Meta conversion can't be applied to those events. You must create a custom conversion using this [Meta guide](https://www.facebook.com/business/help/2375212726097833?id=1205376682832142 "https://www.facebook.com/business/help/2375212726097833?id=1205376682832142").
```

Comment:
`custom data` (or `custom_data` if it's supposed to be an underscore separator). Can also give some more brief information as to what this corresponds to in Meta Pixel.